components/integrations/bots/bots.jsx

```diff
@@ -132,9 +133,9 @@ export default class Bots extends React.PureComponent {
             return React.cloneElement(child, {filter: props.filter});
         });
         return (
-            <div>
+            <>
```

Comment:
Is this valid? Why remove the div?

Reply:
@crspeller this is a new shorthand syntax for react fragments I believe. See https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html

Reply:
Yes, I don't love it, but if we going to use that, should be in a consistent way, I mean, we can decide to change it and use it all-over the application, but until then, we should keep doing in a consistent way, which means `<React.Fragment>`